lib/matplotlib/axes/_axes.py

```diff
@@ -1528,32 +1528,61 @@ def plot(self, *args, scalex=True, scaley=True, data=None, **kwargs):
 
         **Markers**
 
-        =============    ===============================
-        character        description
-        =============    ===============================
-        ``'.'``          point marker
-        ``','``          pixel marker
-        ``'o'``          circle marker
-        ``'v'``          triangle_down marker
-        ``'^'``          triangle_up marker
-        ``'<'``          triangle_left marker
-        ``'>'``          triangle_right marker
-        ``'1'``          tri_down marker
-        ``'2'``          tri_up marker
-        ``'3'``          tri_left marker
-        ``'4'``          tri_right marker
-        ``'s'``          square marker
-        ``'p'``          pentagon marker
-        ``'*'``          star marker
-        ``'h'``          hexagon1 marker
-        ``'H'``          hexagon2 marker
-        ``'+'``          plus marker
-        ``'x'``          x marker
-        ``'D'``          diamond marker
-        ``'d'``          thin_diamond marker
-        ``'|'``          vline marker
-        ``'_'``          hline marker
-        =============    ===============================
+        ============================== ========================================
+        character                      description
+        ============================== ========================================
+        ``'.'``                        point marker
+        ``','``                        pixel marker
+        ``'o'``                        circle marker
+        ``'v'``                        triangle_down marker
+        ``'^'``                        triangle_up marker
+        ``'<'``                        triangle_left marker
+        ``'>'``                        triangle_right marker
+        ``'1'``                        tri_down marker
+        ``'2'``                        tri_up marker
+        ``'3'``                        tri_left marker
+        ``'4'``                        tri_right marker
+        ``'8'``                        octagon marker
+        ``'s'``                        square marker
+        ``'p'``                        pentagon marker
+        ``'P'``                        plus (filled) marker
+        ``'*'``                        star marker
+        ``'h'``                        hexagon1 marker
+        ``'H'``                        hexagon2 marker
+        ``'+'``                        plus marker
+        ``'x'``                        x marker
+        ``'X'``                        x (filled) marker
+        ``'D'``                        diamond marker
+        ``'d'``                        thin_diamond marker
+        ``'|'``                        vline marker
+        ``'_'``                        hline marker
+        ``0`` (``TICKLEFT``)           tickleft marker
```

Comment:
In a format string you can only address markers that are described by a single-character string. So everything below here cannot be used.

Reply:
OK, I have updated it.